metrics/metrics.go

```diff
@@ -38,9 +41,9 @@ func init() {
 
 // NewRegistry creates a new metrics registry.
 func NewRegistry(cfg config.Metrics) (r Registry, err error) {
-	prefix := cfg.Prefix
-	if prefix == "default" {
-		prefix = defaultPrefix()
+
+	if prefix, err = parsePrefix(cfg.Prefix); err != nil {
```

Comment:
As discussed in the thread `default` should still work to avoid breaking existing behavior.